src/components/pages/NewReviewForm.tsx

```diff
@@ -0,0 +1,51 @@
+import Container from '@mui/material/Container'
+import Rating from '@mui/material/Rating'
+import Typography from '@mui/material/Typography'
+import Button from '@mui/material/Button'
+import FormGroup from '@mui/material/FormGroup'
+import { useState } from 'react'
+import { useNavigate } from 'react-router'
+import useCreateReview from '../../hooks/useCreateReview'
+
+export default function NewReview() {
+  // need to send form using useCreateReview hook
+  const navigate = useNavigate()
+  const mutation = useCreateReview()
+  const [form, setForm] = useState<any>({
```

Comment:
do we need this form state here? handleSubmit could get formData and call mutation.mutate with it?

Reply:
I was having trouble doing this on 'Register' page with CreateUser and resorted to using Form state because of the Type issue. I believe when I used FormData, it would not pass the type requirement for 'CreatUser' type that I'm using for Form.  I can revisit and see if I missed something there and pass in FormData directly to mutation.mutate